static/app/views/app/root.tsx

```diff
@@ -18,18 +18,18 @@ import replaceRouterParams from 'sentry/utils/replaceRouterParams';
  * in which case we should load their list of organizations and make a decision
  */
 function AppRoot() {
-  const config = useLegacyStore(ConfigStore);
+  const {lastOrganization} = useLegacyStore(ConfigStore);
 
   useEffect(() => {
-    if (!config.lastOrganization) {
+    if (!lastOrganization) {
       return;
     }
 
-    const orgSlug = config.lastOrganization;
+    const orgSlug = lastOrganization;
```

Comment:
wat? shouldn't this be `lastOrganization.slug`?

Reply:
nah `lastOrganization` is a string